### PR TITLE
Add recentwork modal

### PR DESCRIPTION
**Pull request by Errytagedesign**
# In this branch we did:

- Render work section data from javascript using createElement method and forEach method 
- Create popup functionality
- Create functionality for see project button
- Render each project details when popup is active based on the id of the clicked projects
- Add more styles to style.css

### CODE REVIEW

File: assets/styles/style.css [hunk 1]
```diff
@@ -706,3 +723,149 @@ footer {
...
+}
+
+.project-card {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  background: var(--white);
+  padding: 1.4em 1em;
+  color: var(--black);
+}
```
Comment (taldr27) [inline assets/styles/style.css:752]: - [ ] You did an excellent job with your popup on the mobile version! However, there seems **to be a styling issue when viewing it on the smallest devices,** as the popup gets cut off. Would you mind adjusting the styling using percentages to ensure it displays properly? 😀

   ### Your design:
   image

File: assets/styles/style.css [hunk 2]
```diff
@@ -706,3 +723,149 @@ footer {
...
+  width: 100%;
+  height: 100%;
+  top: 0;
+  padding: 2em 1em;
+  background: #97949371;
+  backdrop-filter: blur(14px);
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
```
Comment (taldr27) [inline assets/styles/style.css:743]: - [ ] You've done a fantastic job with the design in the desktop version! I have a small suggestion to make it look even better. Could you kindly align your buttons to the left, similar to the layout in the Figma template? This adjustment will bring your project closer to the intended design. 🙌

- [ ] Please also consider imcrementing a bit the width of your popup.

   ### Your design:
   image

   ### Figma design:
   image

**Review by taldr27 — CHANGES_REQUESTED** [review]
# Status: Required Changes ♻️♻️
Hello @Errytagedesign, 

Good job so far! 
You're making good progress on the project, but some issues still need to be addressed before you can move on to the next stage. Keep up the good work! 💪

almost-there

## Highlights 🎉
✅ Professional Readme. 
✅ Linter tests passing locally.
✅ Correct use of GitHub Flow.
✅ Great job with your popup window.
✅ Great work using an array of objects for your projects.

## Required Changes ♻️
_Check the comments under the review._

------
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @taldr27 in your question so I can receive the notification.

If you need any further help or have any questions, feel free to reach out to me on Slack as _**Diego Garcia**_.🎯

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s). gif_

**Comment by Errytagedesign** [comment]
Please note that, the description text used in mobile in the figma design isn't same as desktop, and i'm using desktop text across the screen, hence the reasons why i added scroll to the project details card in the pop up.

Just need to clarify that.

When you clicked the see project button, you can scroll up and down to see the hidden part

File: assets/styles/style.css [hunk 3]
```diff
@@ -706,3 +725,153 @@ footer {
...
+.project-card {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  background: var(--white);
+  padding: 1.4em 1em;
+  color: var(--black);
+  width: 95%;
+  height: 100%;
+  overflow: scroll;
```
Comment (Cabraham1) [inline assets/styles/style.css:756]: - [ ] Nice work so far 👍🏿  just as the previous reviewer pointed out, your modal still needs a adjustment with margins and padding, as the button below are cut off from the popup

image

**Review by Cabraham1 — CHANGES_REQUESTED** [review]
# Required Changes :recycle:
almost-there

## Hey @Errytagedesign   :wave:

## Highlights :1st_place_medal:

✔️ Descriptive PR Title
✔️ Descriptive PR Summary
✔️ Following some requirements

Good job on implementing some of the requirements, there are some issues that you still need to work on to go to the next project but you are almost there! :blush:
- [ ] I kindly request that you adhere to the Figma design provided for your project. The Figma design serves as a visual reference and guideline for implementing the desired user interface and overall look of your application. Please ensure that the implemented code aligns with the design specifications provided in Figma. This includes aspects such as layout, color scheme, typography, and any other visual elements defined in the design.
- [ ] Your Design
image

# Optional

_All the changes with optional tags are not crucial enough to prevent you from getting the approval but we highly recommend to students that they should take these changes into consideration._

Cheers, and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

---------------------------------------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: text [hunk 4]
```diff
@@ -0,0 +1,283 @@
...
+            <p id="description">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required. Has been the industry's standard.
+            </p>
+            <ul>
+              <li>
+                <a
+                  href=""
+                  rel="noopener noreferrer"
+                  target="_blank"
```
[the remark this answers is not in the document]
Comment (Errytagedesign) [reply text:13]: What file are you referring to here?

[the remark this answers is not in the document]
Comment (Cabraham1) [reply text]: the text file @Errytagedesign 
image

[the remark this answers is not in the document]
Comment (Errytagedesign) [reply text]: Thank you, i've removed it

**Review by rotimiazeez — APPROVED** [review]
Hi @Errytagedesign 👏🏽, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- Good job implementing the required changes 👏🏾
- Good communication with the CR 👌🏽

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code. Some of them were simply missed by the previous reviewer, and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me(@rotimiazeez) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**